_source/_posts/2019-05-30-vue-pwa-typescript.md

```diff
@@ -357,35 +364,35 @@ In many situations, you will want to restrict access to parts of your applicatio
 
 To start, you need to create an account with Okta. Visit [developer.okta.com](https://developer.okta.com/) and click the **Sign Up** button. On the next screen, enter your details and click on **Get Started**. 
 
-Once you have finished the registration process, you will be taken to the developer dashboard. Each application that you want to use with Okta authentication must be registered and will receive its own client ID. Click on **Add Application** and, on the next screen, select **Single Page Application**. When you click on **Next** you will see a screen with settings. Make sure the port is set to `8080`. This is the port that Vue uses to serve applications.
+Once you have finished the registration process, you will be taken to the developer dashboard. Each application that you want to use with Okta authentication must be registered and will receive its own client ID. Click on **Add Application** and, on the next screen, select **Single Page Application**. When you click on **Next** you will see a screen with settings. Change the default value for **Login redirect URIs** to `http://localhost:8080/callback`. You can leave all the other defaults. After clicking **Done**, scroll down and make note of the **client_id** value. You'll need it shortly.
 
 {% img blog/vue-pwa/create-new-app.png alt:"Create New Okta Application" width:"800" %}{: .center-image }
 
-Once you are finished you will be given a `clientId`. This is needed in your application when configuring Okta. In your application directory now run the following command.
+In your application directory now run the following command.
 
 ```bash
-npm i @okta/okta-vue@1.1.0 @types/okta__okta-vue@1.0.2
+npm i @okta/okta-vue @types/okta__okta-vue
```

Comment:
I would specify versions here - that way when something breaks between major releases, this post will still work.